Patch for new OBR 2.3's new rendering engine

SeamusFinlayson opened the pull request.

SeamusFinlayson committed:
Remove second package manager, node_modules folder

SeamusFinlayson committed:
Fix condition image scaling for new obr rendering engine

SeamusFinlayson committed:
Remove overzealous search refocussing

SeamusFinlayson committed:
Only show focus highlight for keyboard navigation

SeamusFinlayson committed:
Update manifest version

SeamusFinlayson committed:
Merge pull request #1 from SeamusFinlayson/patch-for-new-obr-rendering-engine

Patch for new obr rendering engine

SeamusFinlayson committed:
Merge branch 'main' into main

SeamusFinlayson committed:
Delete node_modules directory for real this time

kamejosh commented on package-lock.json:
I wouldn't delete the package-lock.json if you are not going to replace it with another lock file. having a lock file ensures that every build uses the exact same dependencies and doesn't break because a package is updated.

SeamusFinlayson replied:
There's still a yarn lock in the project, though I'm not sure if @kgbergman uses yarn or npm to deploy so maybe not the best idea anyways.

kamejosh commented on public/manifest.json:
```diff
@@ -1,7 +1,7 @@
-  "version": "1.0.2",
-  "manifest_version": 3,
+  "version": "1.0.3",
+  "manifest_version": 4,
```
I'm not aware that there is a manifest_version beyond `1`

SeamusFinlayson replied:
Oh is that supposed to be for standardizing different versions of the manifest? I usually don't touch it but kgbergman incremented it after they merged my last PR so I figured they wanted it incremented again.

kamejosh replied:
I think this would be used if owlbear would change the way the manifest files work, so they could check the manifest version and know how to parse it. Because there is only one version i think this isn't even checked. I could be wrong though: https://docs.owlbear.rodeo/extensions/reference/manifest

SeamusFinlayson replied:
That makes sense to me. I changed the version to 1 in a new commit.

kamejosh reviewed (commented):
Some suggestions, but otherwise this would be awesome if it could get merged @kgbergman so people could keep using your extension.

SeamusFinlayson committed:
Update manifest.json

Owlbear Rodeo has not made updates to the manifest formatting so 1 should be used as it is in the example projects.

SeamusFinlayson commented:
I'm resubmitting this pr with a branch from my fork
